modules/google-cert/variables.tf

```diff
@@ -0,0 +1,12 @@
+variable "domains" {
```

Comment:
We need a README file for this module, explaining its purpose and giving a simple usage example.